country-a-service/testing-shared/src/main/java/dk/sundhedsdatastyrelsen/ncpeh/testing/shared/FmkResponseStorage.java

```diff
@@ -16,13 +20,18 @@
 import java.nio.file.Path;
 import java.util.List;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 /**
  * These methods are concerned with retrieving and serializing responses (ePrescriptions) from FMK
  * so that they can be used as reliable test data for e.g. CDA generation.
  */
 public class FmkResponseStorage {
+    private static final Logger logger = LoggerFactory.getLogger(FmkResponseStorage.class);
```

Comment:
Have you checked that this works when the main class is run in IntelliJ?  If not, then I suggest sticking with System.out.println.